brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced columns - Resolved width not setup on responsive devices in editor.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/23ece378-83a0-4609-a26e-a1e70ee40ea8?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots

- Resolved screencast - https://share.getcloudapp.com/JruOwLkR

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the Advanced column.
- Set up different widths on desktop, tablet, and mobile.
- Check by switching to preview on tablet and mobile.

### Checklist:
